app/models/service.rb

```diff
@@ -97,11 +97,11 @@ def direct_vms
   end
 
   def all_vms
-    MiqPreloader.preload_and_map(subtree, :direct_vms)
+    MiqPreloader.preload_and_map(subtree, :vms)
```

Comment:
@kbrock direct_vms has one problem, I have service_orchestration, that has VMs stored under nested orchestration stacks. 

So calling ServiceOrchestration.find(xy).direct_vms will not load any VMs, since I need to really call ServiceOrchestration.find(xy).all_vms

I added method vms, cause calling all_vms would lead to infinite cycle (subtree contains itself)

---

I need to circle back to this, to check if there was a n+1 before